packages/dashboard-core-plugins/src/panels/ConsolePanel.jsx

```diff
@@ -302,6 +303,18 @@ export class ConsolePanel extends PureComponent {
                 <>
                   <div>&nbsp;</div>
                   <div>{ConsoleConstants.LANGUAGE_MAP.get(language)}</div>
+                  <div>&nbsp;</div>
+                  <div>
+                    <SizeUsageUI
+                      connection={connection}
+                      UIParams={{
+                        defaultUpdateInterval: 10000,
+                        hoverUpdateInterval: 3000,
+                      }}
+                      monitorDuration={60 * 10}
```

Comment:
I would flip the math here for readability. `10 * 60` since it's 10 minutes and then you're multiplying by 60 as a conversion to seconds. So for millis then `10 * 60 * 1000`

Basically the 10 is what we care about, but reading the code we see 60 first